models/published/direct_care/olids/covid_flu/covid_flu_dashboard_base.sql

```
@@ -56,11 +65,22 @@ WITH uptake_with_demographics AS (
+        WHEN d.IMD_QUINTILE_25 = 'Second Most Deprived' THEN 2
+        WHEN d.IMD_QUINTILE_25 = 'Third Most Deprived' THEN 3
+        WHEN d.IMD_QUINTILE_25 = 'Second Least Deprived' THEN 4
+        WHEN d.IMD_QUINTILE_25 = 'Least Deprived' THEN 5
+        ELSE 6 END AS imdquintile_order,
         d.lsoa_code_21 as lsoa_code,
         d.lsoa_name_21 as lsoa_name,
-        d.borough_resident,
+        d.ward_code,
+        d.ward_name,
+        COALESCE(la.LAD25_NM,'Unknown') AS borough_resident,
+        CASE WHEN la.RESIDENT_FLAG IS NULL THEN 'Unknown'
```

SQLFluff (4.2.2)

[error] Boolean/null literals must be lower case.

(CP04)